core/src/i18n/en.yaml

```
@@ -670,13 +672,15 @@ containers:
     invalidimagename: Invalid image name
     networkpluginnotready: Network plugin not ready
     no-restarts: Container has not restarted
+    pending: Container scheduled
     podinitializing: Initializing the Pod
     poststarthookerror: Post-start hook error
     registryunavailable: Registry unavailable
     runcontainererror: Run container error
     running: Container running correctly
     terminated: Container terminated
     terminating: Terminating the Container
+    unknown: Container state is unknown
```

```suggestion
    unknown: Container state unknown
```
Since it's tooltip let's keep it simple.